Configurations/windows-makefile.tmpl

```diff
@@ -425,8 +427,8 @@ install_dev:
 uninstall_dev:
 
 install_engines:
-	@if "$(INSTALLTOP)"=="" ( echo INSTALLTOP should not be empty & exit 1 )
-	@echo *** Installing engines
+	@if "$(INSTALLTOP)"=="" ( $(ECHO) "INSTALLTOP should not be empty" & exit 1 )
```

Comment:
On side note one doesn't have to replace *this* echo. Because it's beyond nmake's control. I mean nmake will have to pass this whole line to cmd.

Reply:
Ooooooh! It first looks for if.com, if.exe, if.bat, if.vbs, if.js, etc. before it decides to pass it to cmd. This actually feels like a bug...

Reply:
And if I try to (if ...), it look for (if.com, (if.exe, etc. It's just no escape...

Reply:
There is an escape! And if spells white space. I mean if there space right after @, then cmd is called. But it's not general rule. In sense that if I type `@ echo`, it still looks for echo.com, echo.exe, etc...

Reply:
> There is an escape!

Or not. It's actually not white space but a character that would be viewed by nmake as shell. Most notably & triggers call to cmd. So that one can shortcut `@rem ...` as `@rem & ...` One can also shortcut `@echo something` as `@echo something & :`